### PR TITLE
PAYARA-1182 Created stop-all-domains command

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
jenkins test please

**Comment by Cousjava** [comment]
jenkins test please

**Comment by mulderbaba** [comment]
jenkins test please

**Comment by mulderbaba** [comment]
jenkins test please

File: nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/cli/StopDomainCommand.java [hunk 1]
```diff
@@ -88,11 +88,12 @@ protected void initDomain() throws CommandException {
         // NetUtils says that the getHost() --> isThisMe() rather than merely
         // checking for the magic "localhost" string.  Too risky to fool with it today.
 
-        if (programOpts.getHost().equals(CLIConstants.DEFAULT_HOSTNAME))
+        if (programOpts.getHost().equals(CLIConstants.DEFAULT_HOSTNAME)) {
```
Comment (smillidge) [inline nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/cli/StopDomainCommand.java]: You need to add the comment that you have changed the file into the header like;
// Portions Copyright [2017] [Payara Foundation and/or its affiliates]

File: nucleus/admin/server-mgmt/src/main/java/fish/payara/admin/servermgmt/cli/StopAllDomainsCommand.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
+/*
+
```
Comment (smillidge) [inline nucleus/admin/server-mgmt/src/main/java/fish/payara/admin/servermgmt/cli/StopAllDomainsCommand.java]: This is the incorrect header pleas
```java
 * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
 *
 * Copyright (c) [2017] Payara Foundation and/or its affiliates. All rights reserved.
 *
 * The contents of this file are subject to the terms of either the GNU
 * General Public License Version 2 only ("GPL") or the Common Development
 * and Distribution License("CDDL") (collectively, the "License").  You
 * may not use this file except in compliance with the License.  You can
 * obtain a copy of the License at
 * https://github.com/payara/Payara/blob/master/LICENSE.txt
 * See the License for the specific
 * language governing permissions and limitations under the License.
 *
 * When distributing the software, include this License Header Notice in each
 * file and include the License file at glassfish/legal/LICENSE.txt.
 *
 * GPL Classpath Exception:
 * The Payara Foundation designates this particular file as subject to the "Classpath"
 * exception as provided by the Payara Foundation in the GPL Version 2 section of the License
 * file that accompanied this code.
 *
 * Modifications:
 * If applicable, add the following below the License Header, with the fields
 * enclosed by brackets [] replaced by your own identifying information:
 * "Portions Copyright [year] [name of copyright owner]"
 *
 * Contributor(s):
 * If you wish your version of this file to be governed by only the CDDL or
 * only the GPL Version 2, indicate your decision by adding "[Contributor]
 * elects to include this software in this distribution under the [CDDL or GPL
 * Version 2] license."  If you don't indicate a single choice of license, a
 * recipient has the option to distribute your version of this file under
 * either the CDDL, the GPL Version 2 or to extend the choice of license to
 * its licensees as provided above.  However, if you add GPL Version 2 code
 * and therefore, elected the GPL Version 2 license, then the option applies
 * only if the new code is made subject to such option by the copyright
 * holder.
 */
```

**Comment by Cousjava** [comment]
jenkins test please

File: nucleus/admin/server-mgmt/src/main/java/fish/payara/admin/servermgmt/cli/StopAllDomainsCommand.java (hunk 2 above)
Comment (Cousjava) [inline nucleus/admin/server-mgmt/src/main/java/fish/payara/admin/servermgmt/cli/StopAllDomainsCommand.java]: done change

**Pull request merged**